releasenotes/notes/add-subgraph-replace-ec54792df1d641fb.yaml

```
@@ -0,0 +1,34 @@
+---
+features:
+  - |
+    A new method, :meth:`~retworkx.PyDiGraph.substitute_node_with_subgraph`,
+    to the :class:`~retworkx.PyDiGraph` class. This method is used to replace a
+    node in a :class:`~retworkx.PyDiGraph` object with another
+    :class:`~retwork.PyDiGraph` object. For example, first creating a new
+    graph:
+
+    .. jupyter-execute::
+
+      import retworkx
+      from retworkx.visualization import mpl_draw
+
+      original_graph = retworkx.generators.directed_path_graph(5)
+      mpl_draw(original_graph)
```

Because the feature refers to node labels, I would include the labels in this drawing.

```suggestion
      mpl_draw(original_graph, with_labels=True)
```